angular-material-calendar.css

```diff
@@ -50,3 +50,5 @@ calendar-md {
       calendar-md md-content > md-content.calendar > :not(:first-child) > * :last-child {
         overflow: hidden;
         text-overflow: ellipsis; }
+      calendar-md md-content > md-content.calendar .currentDayHighlight {
+          background-color: #FCF8CD !important; }
```

Comment:
Can we please not use the !important here? I don't see anything else setting the background color property so there should be no need for using !important.